_pages/about-us/teams/products-and-platforms.md

```diff
@@ -1,10 +1,12 @@
 ---
 title: Products and Platforms
 ---
-Products and Platforms is one of the 5 business units within 18F. We strive to incubate tools for digital government.
```

Comment:
> We help our agency partners conceptualize and build products for digital government.

This doesn't work. Federalist, login.gov, cloud.gov, eRegs, and the design standards are all intended to enable our partners to serve the public better. We don't build products for our partners; instead, our partners use our products and platforms to deliver on their mission more efficiently.